Fixed srf filtering (#429)
eradiate/eradiate: docs/src/release_notes/v0.28.x.md

* Added moment integrator and radiance variance output ({ghpr}`426`).

% ### Changed
### Fixed

> * Fixed a bug in the SRF filtering tool ({ghpr}`429`).

Reviewer: ```suggestion
* Fixed out-of-bounds indexing in the SRF filtering tool ({ghpr}`429`).
```
